README.rst

```diff
@@ -1,2 +1,95 @@
-layered-config
+layered_config
 ==============
+
+A simple library to ease maintaining groups of similar configurations:
+   * Keep config files DRY like code.
+   * Allow environment variables to override configuration file values.
+   * `munch`_ your loaded configuration for easy field access.
+
+Instead of copy'n'paste-ing related configuration files for different situations,
+refactor the configuration sections and keys into separate smaller configuration files.
+Then use this library to combine those files in various ways to produce a single |ConfigParser| object,
+as if it had come from a single file.
+
+
+tl;dr Quick Start in 3 Easy Steps:
+----------------------------------
+
+1. Create some configs:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+For this simple example, consider three config files:
```

Comment:
Only asking because I've genuinely forgotten the answer to this question myself:

Does it matter if all of the cake config layers are in the same directory? Regardless, is it worth mentioning?

Reply:
Thanks a good question, they don't, and I can update the example and add a brief note explaining. Nice catch.

Reply:
Cool. :) Tag me in this comment when you've done that and I'll come back and approve.